widgets/features/features.php

```diff
@@ -194,14 +194,14 @@ function get_widget_form(){
 
 			'container_shape' => array(
 				'type' => 'select',
-				'label' => __('Container shape', 'so-widgets-bundle'),
+				'label' => __('Icon Container Position', 'so-widgets-bundle'),
```

Comment:
Looks like this was accidentally changed from _shape_ to _position_ and sentence case: _Icon container shape_